### PR TITLE
Change Default Configuration, Rename Signal

### PR DESCRIPTION
**Pull request by jhazentia**
- Change default configuration from USRP to Tektronix with preselector.
- `monitor_action_completed` signal renamed to `trigger_api_restart`
- updated dependencies
- added `"django.template.context_processors.request"` to remove docker warning
- other misc changes

### CODE REVIEW

File: configs/preselector_config.json [hunk 1]
```diff
@@ -1 +1,14 @@
-{}
+{
+	"name": "preselector",
```
Comment (aromanielloNTIA) [inline configs/preselector_config.json:2]: We may not want to include this file, or the calibration JSON files, or the sensor_definition JSON file.

File: docker-compose.yml [hunk 2]
```diff
@@ -14,9 +14,10 @@ services:
   api:
     healthcheck:
       test: test ! -e /sdr_unhealthy && test ! -e /scheduler_dead
-      interval: 10s
+      interval: 6s
       timeout: 3s
       retries: 1
+    shm_size: '16gb'
```
Comment (aromanielloNTIA) [inline docker-compose.yml:20]: This shared memory configuration may not be best to include all the time- this is chosen specifically for SEA NUCs with 32GB total RAM. Only the SEA action makes use of shared memory at all.

File: docs/openapi.json [hunk 3]
```diff
@@ -1860,7 +1862,7 @@
                 "duration": {
                     "title": "Duration",
                     "description": "Task duration in seconds",
-                    "type": "number"
+                    "type": "string"
```
Comment (aromanielloNTIA) [inline docs/openapi.json:1865]: Why is this changed from a number to a string?

File: scripts/rebuild.sh [hunk 4]
```diff
@@ -0,0 +1,6 @@
+docker-compose down
```
Comment (aromanielloNTIA) [inline scripts/rebuild.sh:1]: This is a script which I have found very useful in testing, but could be dangerous to run without knowing its purpose. We might consider excluding this file.

File: src/sensor/settings.py [hunk 5]
```diff
@@ -366,6 +367,7 @@
         "tasks": {"handlers": ["console"], "level": LOGLEVEL},
         "scos_actions": {"handlers": ["console"], "level": LOGLEVEL},
         "scos_usrp": {"handlers": ["console"], "level": LOGLEVEL},
+        "scos_tekrsa": {"handlers": ["console"], "level": LOGLEVEL},
         "scos_sensor_keysight": {"handlers": ["console"], "level": LOGLEVEL},
         "scos_tekrsa": {"handlers": ["console"], "level": LOGLEVEL},
```
Comment (aromanielloNTIA) [inline src/sensor/settings.py:372]: `scos_tekrsa` is included twice here. Also `scos_usrp` is still included (although this is probably OK)

File: docs/openapi.json (hunk 3 above)
Comment (jhazentia) [reply docs/openapi.json:1865]: I'm not sure, this file is automatically generated

File: configs/preselector_config.json (hunk 1 above)
Comment (aromanielloNTIA) [reply configs/preselector_config.json]: Emptied the preselector config file in [aad728d](https://github.com/NTIA/scos-sensor/pull/218/commits/aad728dd792f2c0ad4755319bffc56a3840cacd2).

Deleted calibration files in [c753810](https://github.com/NTIA/scos-sensor/pull/218/commits/c7538108dc164b5538a345ad7cdd27079b7c4104).

Sensor definition changes are OK.

File: scripts/rebuild.sh (hunk 4 above)
Comment (aromanielloNTIA) [reply scripts/rebuild.sh]: Removed this script in [b7aac5e](https://github.com/NTIA/scos-sensor/pull/218/commits/b7aac5eef24e809ef7ca12d371bf7d930189addc) since it is potentially dangerous and very easy to re-create locally when needed. It is also not a useful script in production.

File: docs/openapi.json (hunk 3 above)
Comment (aromanielloNTIA) [reply docs/openapi.json:1865]: Confirmed that this is actually the correct type - the 'duration' reported in the task results page is a string, e.g. "00:00:58.796552".

The description, however, is incorrect, and should be changed to something like "Task duration, in hh:MM:SS.ss format"

File: src/sensor/settings.py (hunk 5 above)
Comment (aromanielloNTIA) [reply src/sensor/settings.py]: Removed new/redundant entry in [027c83e](https://github.com/NTIA/scos-sensor/pull/218/commits/027c83edd17c1ef51e5d4c1a7d100cf2617e3c81). Leaving `scos_usrp` and `scos_sensor_keysight` is fine here.

File: docker-compose.yml (hunk 2 above)
Comment (jhazentia) [reply docker-compose.yml:20]: I added a section to the README about this, per Doug's suggestion.

File: docs/openapi.json (hunk 3 above)
Comment (jhazentia) [reply docs/openapi.json:1865]: I updated the description, now says: "Task duration, in %H:%M:%S.%f format"

This is the Python format from DATETIME_FORMAT setting

**Review by aromanielloNTIA — APPROVED** [review]
